pkg/cmd/roachtest/roachtestutil/utils.go

```diff
@@ -65,6 +67,49 @@ func (e *EveryN) ShouldLog() bool {
 	return e.ShouldProcess(timeutil.Now())
 }
 
+// GetWorkloadHistogramArgsString creates a histogram flag string based on the roachtest to pass to workload binary
+// This is used to make use of t.ExportOpenmetrics() method and create appropriate exporter
+func GetWorkloadHistogramArgsString(
```

Comment:
nit: Maybe just `GetWorkloadHistogramArgs`? IMO you can drop `String` since it's implied by `Args`.